client/src/components/Header.js

```diff
@@ -7,17 +7,18 @@ import Title from '../components/Title';
 const HeaderContainer = styled.div`
   display: flex;
   justify-content: space-between;
+  align-items: center;
   position: fixed;
-  width: 100%;
-  height: 10%;
+  width: 100vw;
+  height: 10vh;
```

Comment:
probably better to use a fixed height instead of the `10vh`, something like 48 or 56 pixel maybe

Reply:
<img width="486" alt="Screenshot 2020-04-17 at 12 20 53" src="https://user-images.githubusercontent.com/60884180/79559456-0da40180-80a6-11ea-83fe-aa5edfbd35cf.png">

I tried, but then the spacing of the blur image in either the mobile or desktop view no longer fits. But I'll keep it in mind and try again at a later stage, thanks!